ouroboros-consensus/src/Ouroboros/Consensus/Ledger/Dual.hs

```diff
@@ -427,6 +432,13 @@ instance Bridge m a => TableStuff (LedgerState (DualBlock m a)) where
        foldLedgerTables f main
     <> foldLedgerTables f aux
 
+  foldLedgerTables2
```

Comment:
In general, both this new `foldLedgerTables2` and the existing `foldLedgerTables` definition might be wrong for `DualBlock`. Mainly: should the result include the `aux` related things or not? I don't think any other function so directly mixes the `main` and `aux` bits 🤔

Reply:
We talked about this and agreed that the plan seems reasonable.

My TODO: I'll create a ticket.